package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "dwitter",
   "version": "1.0.0",
-  "description": "Social network for short js demos",
+  "description": "A twitter-style network of 140-byte javascript canvas demos",
```

Comment:
140-character ;)

Reply:
Of course *facepalm*... I keep getting confused cause Discord people keep calling it that.